tests/bootstrap.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace RevAPI;
+
+$autoloader = require __DIR__ . '/../vendor/autoload.php';
+$autoloader->add(__NAMESPACE__, __DIR__);
+
+if (file_exists(__DIR__ . '/config.inc.php')) {
+    $config = require_once __DIR__ . '/config.inc.php';
```

Comment:
`require_once` in this case is not a good idea. What if some other part of the app needed to load that configuration file (i know it doesn't)? The `_once` part of these PHP constructs is to prevent side-effects, but we already know/expect that file to be side-effect free.